src/Components/ContextualMenu/__stories__/ContextualMenu.story.tsx

```diff
@@ -107,14 +107,14 @@ export const RichContent = () => {
 
   return (
     <ContextualMenu>
-      <ContextualMenuItem onClick={confirmIt}>
+      <ContextualMenuItemButton onClick={confirmIt}>
```

Comment:
The switch from `ContextualMenuItem` to `ContextualMenuItemButton` has had a funny layout side effect on this `RichContent` story:

| Before | After |
|--------|--------|
|  <img width="400" alt="b444" src="https://github.com/user-attachments/assets/b09882e6-75d3-4878-929c-52377a3ad4c5" /> | <img width="400" alt="a444" src="https://github.com/user-attachments/assets/78a8a407-0fb8-4048-902d-e84429d020d5" /> |

I don't think the component has been used with rich content in the menu items, so it's not the highest priority. But wondering if it's an easy fix.

Reply:
Ah woops my bad. Didn't check the stories 🙄 Will certainly fix.